src/cjson/lua_cjson.h

```diff
@@ -0,0 +1,8 @@
+#ifndef NVIM_LUA_CJSON_H
+#define NVIM_LUA_CJSON_H
```

Comment:
the header guards should be renamed, `NVIM_` prefix doesn't match their path.

Reply:
Can you clarify what you would like the guard to be? `CJSON_H`?

Reply:
`CJSON_LUACJSON_H` i guess

Reply:
Ah, got it. Done.